content/committees/steering/_index.md

```diff
@@ -3,4 +3,20 @@ title: "Steering Committee"
 date: 2020-01-05T15:50:25-04:00
 ---
 
-This committee owns the overall strategy and the high level, long term roadmap spanning across multiple releases, produces the Jakarta EE release key messaging and gets periodic updates from other committees and Jakarta EE Platform Project on the status of the release and any related activities.
\ No newline at end of file
+This committee owns the overall strategy and the high level, long term roadmap spanning across multiple releases, produces the Jakarta EE release key messaging and gets periodic updates from other committees and Jakarta EE Platform Project on the status of the release and any related activities.
+
+<!--more-->
+
+{{< meeting-minutes-single "steering_committee" >}}
+
+### Committee Members
+
+* Will Lyons (**chair**)- Oracle, Ed Bratt - alternate
+* Kenji Kazumura - Fujitsu, Michael DeNicola - alternate
+* Dan Bandera - IBM, Ian Robinson - alternate
+* Steve Millidge - Payara, Eliot Martin - alternate
+* Mark Little - Red Hat, Scott Stark - alternate
+* David Blevins - Tomitribe, Cesar Hernandez - alternate
+* Martijn Verburg - London Java Community - Elected Participant Representative
+* Ivar Grimstad - Elected Committer Representative
```

Comment:
Arjan Tijms is the Elected Committer Representative

Reply:
Done! thx